src/core/arith.rs

```diff
@@ -378,13 +378,13 @@ pub trait Convertable {
 
 impl<T> Convertable for T
 where
-    T: Copy + HasAfEnum + ConstGenerator<OutType = T>,
+    T: Clone + HasAfEnum + ConstGenerator<OutType = T>,
```

Comment:
I am curious as to how this will work ? Especially when one of the parameters to overloaded function is an `Array<>` because `Array` doesn't need to satisfy `HasAfEnum` or `ConstGenerator` traits.

Reply:
It would just fail, hm.